### PR TITLE
Fixed Authority Logs Typo

### PR DESCRIPTION
**Pull request by Robbie-Microsoft**

### CODE REVIEW

File: lib/msal-common/src/authority/Authority.ts [hunk 1]
```diff
@@ -482,7 +482,7 @@ export class Authority {
         }
         
         // Metadata could not be obtained from config, cache or network
-        this.logger.verbose("Did not find cloud discovery metadata from hardcoded values... Metadata could not be obtained from config, cache, network or hardcoded value. Throwing Untrusted Authority Error.");
+        this.logger.verbose("Did not find cloud discovery metadata from hardcoded values... Metadata could not be obtained from config, cache, network or hardcoded values. Throwing Untrusted Authority Error.");
```
Comment (bgavrilMS) [inline lib/msal-common/src/authority/Authority.ts]: this should be logged with `error` verbosity

**Review by bgavrilMS — APPROVED** [review]

File: change/@azure-msal-common-053b4c5b-c11f-4f09-b665-e4bf38740e9c.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```
Comment (tnorling) [inline change/@azure-msal-common-053b4c5b-c11f-4f09-b665-e4bf38740e9c.json]:
```suggestion
  "type": "patch",
```

**Review by tnorling — APPROVED** [review]

File: lib/msal-common/src/authority/Authority.ts (hunk 1 above)
Comment (Robbie-Microsoft) [reply lib/msal-common/src/authority/Authority.ts]: good catch

File: change/@azure-msal-common-053b4c5b-c11f-4f09-b665-e4bf38740e9c.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  "comment": "Fixed Typo in Authority Logs",
```
Comment (sameerag) [inline change/@azure-msal-common-053b4c5b-c11f-4f09-b665-e4bf38740e9c.json]:
```suggestion
  "comment": "Fixed Typo in Authority Logs (#5430)",
```

**Review by sameerag — APPROVED** [review]

**Pull request merged**